data-prepper-plugins/event-json-codecs/src/test/java/org/opensearch/dataprepper/plugins/codec/event_json/EventJsonInputCodecTest.java

```diff
@@ -70,15 +74,15 @@ public void inCompatibleVersionTest() throws Exception {
         final String key = UUID.randomUUID().toString();
         final String value = UUID.randomUUID().toString();
         Map<String, Object> data = Map.of(key, value);
-        Instant startTime = Instant.now();
```

Comment:
In later versions of Java on Linux, there is an issue where the nanosecond resolution results in test failures.